101-azure-machine-learning-service-workspace/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
format the json (e.g. SHIFT+ALT+F in VS Code)

Reply:
done, though some files have spacing of 4 and one has a spacing of 2. Does that matter?